src/views/Item/ItemOKRs.vue

```diff
@@ -154,7 +154,26 @@ export default {
 
   created() {
     if (this.filteredPeriods.length > 0) {
-      this.setPeriod(this.filteredPeriods[0].id);
+      let defaultPeriodIndex = 0;
+      const now = new Date();
+
+      for (const [index, period] of this.filteredPeriods.entries()) {
+        // Set default tab to most recent, but active, period (if any)
+        if (
+          period.startDate &&
+          period.endDate &&
+          isWithinInterval(now, {
+            start: period.startDate.toDate(),
+            end: period.endDate.toDate(),
+          })
+        ) {
+          defaultPeriodIndex = index;
+          break;
+        }
+      }
```

Comment:
Kunne eventuelt vært trukket ut i en hjelpefunksjon eller computed for å slippe den `let`/`break`-logikken?